src/webview/welcome/app/welcome.scss

```diff
@@ -0,0 +1,353 @@
+@import './base';
+@import './buttons';
+@import './icons';
+@import './utils';
+
+body {
+    &.vscode-light {
+        background-color: var(--color-background--darken-05);
+    }
+}
+
+header {
+    display: grid;
+    grid-template-columns: max-content minmax(396px, auto);
+    grid-gap: 1em 4em;
+    align-items: center;
+    margin: 0 2em;
+
+    @media all and (max-width: 768px) {
+        grid-template-columns: auto;
+        justify-items: center;
+        grid-gap: 1rem;
+    }
+}
+
+.container {
+    margin: 1em auto;
+    max-width: 1200px;
+    min-width: 450px;
+}
+
+.content__area {
+    font-size: 1.4rem;
+
+    .vscode-dark & {
+        background-color: var(--color-background--lighten-05);
+    }
+
+    .vscode-light & {
+        background-color: var(--color-background);
+    }
+
+    @media all and (max-width: 768px) {
+        grid-column: span 1;
+    }
+}
+
+.content__area--full-scroll {
+    background-color: unset !important;
+    margin-bottom: 90vh;
+
+    .section--settings {
+        margin: 0 0 1em 0;
+    }
+}
+
+.header__logo {
+    display: flex;
+    flex-wrap: nowrap;
+}
+
+.header__title {
+    font-family: var(--vscode-font-family);
+    color: #EE0000;
```

Comment:
@msivasubramaniaan Make it white color so that it goes with the VSCode Theme. The Red Color is not needed here.